common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -41,9 +41,15 @@ class WordCloudFields(object):
         scope=Scope.settings,
         default="Word cloud"
     )
+    instructions = String(
+        display_name=_("Instructions"),
+        help=_("Add instructions for this word cloud"),
+        scope=Scope.settings,
```

Comment:
It feels like this should really be Scope.content (though perhaps it is arguably the same as display name). In general the line between Scope.content and Scope.settings is very murky, so I'm OK with all of these being Scope.settings.